generic/tests/cfg/nfs_corrupt.cfg

```diff
@@ -16,9 +16,8 @@
     post_command_noncritical = yes
     wait_paused_timeout = 240
     nfs_stat_chk_re = "running"
-    s390x:
-        bootindex_image1 = 0
-        bootindex_stg = 1
+    serial = TARGET_DISK
```

Comment:
serial is a common option for some devices, better to add a prefix.

Reply:
agree

Reply:
updated